zanata-war/src/main/webapp-jboss/WEB-INF/classes/META-INF/persistence.xml

```diff
@@ -87,6 +87,8 @@
       <!-- Search engine -->
       <property name="hibernate.search.default.directory_provider"
         value="org.hibernate.search.store.impl.FSDirectoryProvider" />
+      <property name="hibernate.search.default.locking_strategy"
+        value="simple" />
```

Comment:
Apparently, as of Hibernate Search 4.1, the default is native.

And going by our production logs, we have been using NativeFSLock:

> (Hibernate Search: Index updates queue processor for index org.zanata.model.HTextFlowTarget-1) [] HSEARCH000058: HSEARCH000117: IOException on the IndexWriter: org.apache.lucene.store.LockReleaseFailedException: Cannot forcefully unlock a NativeFSLock which is held by another indexer component: /var/lib/zanata/index/org.zanata.model.HTextFlowTarget/write.lock